.drone.star

```diff
@@ -45,6 +45,7 @@ config = {
     "phpstan": True,
     "phan": True,
     "php-unit": True,
+    "ocisBranches":["master","stable"]
```

Comment:
looks like there's no starlark linter. Would be great to have it as CI pipeline
CC @amrita-shrestha 
```suggestion
    "ocisBranches":["master", "stable"]
```